ui/app/src/components/WorkflowCancellationDialog/utils.ts

```diff
@@ -22,7 +22,6 @@ import { EnhancedDialogState } from '../../hooks/useEnhancedDialogState';
 
 export interface WorkflowCancellationDialogUIProps {
   items: SandboxItem[];
-  classes?: any;
```

Comment:
The class name props should stay. This one in particular should be typed correctly.